omnibus/config/software/libjemalloc.rb

```diff
@@ -0,0 +1,42 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https:#www.datadoghq.com/).
+# Copyright 2022-present Datadog, Inc.
+
+name "libjemalloc"
+default_version "5.3.0"
+
+version "5.3.0" do
+  source sha256: "ef6f74fd45e95ee4ef7f9e19ebe5b075ca6b7fbe0140612b2a161abafb7ee179" 
+end
+
+ship_source_offer true
+
+source url:"https://github.com/jemalloc/jemalloc/archive/refs/tags/#{version}.tar.gz",
+       extract: :seven_zip
+
+relative_path "libjemalloc"
+
+build do
+    license "BSD-2-Clause"
+    license_file "./COPYING"
+
+    env = with_standard_compiler_flags
+
+    # By default jemalloc releases pages using madvise with MADV_FREE which tells the
+    # OS to reclaim pages lazily.
+    # This can result in misleading metrics, so we need to disable it here.
+    # https://github.com/jemalloc/jemalloc/blob/dev/INSTALL.md#advanced-configuration
+    command "echo \"je_cv_madv_free=no\" > config.cache"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:**  Does this look correct?‏

Reply:
This will not be taken into account by `configure` without the `-C` option.

However you should be able to add this to the environment and configure should pick it up

Reply:
I had missed @chouquette's comment. Would that work by simply adding `-C` to the configure options below?

Reply:
I've (hopefully correctly) set it as an environment variable instead..